packages/gatsby-transformer-asciidoc/changelog.md

```diff
@@ -0,0 +1,10 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+<a name="2.0.0-alpha.1"></a>
+
+## 2.0.0-alpha.1 (2018-10-07)
+
+Initial Commit.
```

Comment:
Let's just remove this file completely - `lerna` generates changelog automatically when publishing

Reply:
Ahhh. I wasn't familiar with Lerna. I just emulated another plugin best I could.